### PR TITLE
Add signals for files move/delete/rename to FileSystemDock

### PR DESCRIPTION
**Pull request by groud**
As asked by @reduz  and @neikeq if I am not mistaking.

### CODE REVIEW

File: editor/filesystem_dock.cpp [hunk 1]
```diff
@@ -2303,6 +2317,11 @@ void FileSystemDock::_bind_methods() {
 
 	ADD_SIGNAL(MethodInfo("instance", PropertyInfo(Variant::POOL_STRING_ARRAY, "files")));
 	ADD_SIGNAL(MethodInfo("open"));
+
+	ADD_SIGNAL(MethodInfo("file_removed", PropertyInfo(Variant::STRING, "file")));
+	ADD_SIGNAL(MethodInfo("folder_removed", PropertyInfo(Variant::STRING, "folder")));
+	ADD_SIGNAL(MethodInfo("file_moved_or_renamed", PropertyInfo(Variant::STRING, "old_file"), PropertyInfo(Variant::STRING, "new_file")));
+	ADD_SIGNAL(MethodInfo("folder_moved_or_renamed", PropertyInfo(Variant::STRING, "old_folder"), PropertyInfo(Variant::STRING, "new_file")));
```
Comment (neikeq) [inline editor/filesystem_dock.cpp]: I would just name them `file_moved` and `folder_moved` as it can be interpreted as moving from one name to another.

**Comment by neikeq** [comment]
Thanks! :)

File: editor/filesystem_dock.cpp (hunk 1 above)
Comment (groud) [reply editor/filesystem_dock.cpp]: IMHO it is clearer this way. 
I don't have a strong opinion on it, so I'd like to know if other people agree. :)

Comment (akien-mga) [reply editor/filesystem_dock.cpp]: I also prefer only `file_moved`, "moved" and "renamed" are the same operation (hence why you rename files with `mv` on Unix).

**Comment by groud** [comment]
Ok, I surrendered to the doxa and changed the signals' names. ^^

**Pull request merged**